src/Features/Core/Portable/UnsealClass/AbstractUnsealClassCodeFixProvider.cs

```diff
@@ -0,0 +1,88 @@
+        }
+
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var syntaxRoot = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            var semanticModel = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
+
+            var node = syntaxRoot.FindNode(context.Span, getInnermostNodeForTie: true);
+
+            if (semanticModel.GetSymbolInfo(node, context.CancellationToken).Symbol is INamedTypeSymbol type &&
+                type.TypeKind == TypeKind.Class && type.IsSealed && !type.IsStatic)
+            {
+                var definition = await SymbolFinder.FindSourceDefinitionAsync(
+                    type, context.Document.Project.Solution, context.CancellationToken).ConfigureAwait(false);
+                if (definition != null && definition.DeclaringSyntaxReferences.Length > 0)
+                {
+                    context.RegisterCodeFix(
+                        new MyCodeAction(
+                            c => UnsealDeclarationsAsync(context.Document.Project.Solution, definition.DeclaringSyntaxReferences, c),
+                            string.Format(TitleFormat, type.Name)),
+                        context.Diagnostics);
+                }
+            }
+        }
+
+        private static async Task<Solution> UnsealDeclarationsAsync(
+            Solution solution, ImmutableArray<SyntaxReference> declarationReferences, CancellationToken cancellationToken)
+        {
+            foreach (var (documentId, syntaxReferences) in
+                declarationReferences.GroupBy(reference => solution.GetDocumentId(reference.SyntaxTree)))
+            {
+                var document = solution.GetDocument(documentId);
+                var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+
+                var generator = SyntaxGenerator.GetGenerator(document);
+                var editor = new SyntaxEditor(root, generator);
+
+                foreach (var syntaxReference in syntaxReferences)
+                {
+                    var declaration = await syntaxReference.GetSyntaxAsync(cancellationToken).ConfigureAwait(false);
+
+                    var modifiers = generator.GetModifiers(declaration);
+                    if (modifiers.IsSealed)
+                    {
+                        var newDeclaration = generator.WithModifiers(declaration, modifiers.WithIsSealed(false));
+
+                        editor.ReplaceNode(declaration, newDeclaration);
+                    }
+                }
+
+                solution = solution.WithDocumentSyntaxRoot(documentId, editor.GetChangedRoot());
+            }
+
+            return solution;
+        }
+
+        private sealed class MyCodeAction : CodeAction.SolutionChangeAction
+        {
+            public MyCodeAction(Func<CancellationToken, Task<Solution>> createChangedSolution, string title)
```

Comment:
take title first to match base constructor.

Reply:
➡️ Fixed in 644aaa3d2ca272f59e78587ada2bde5d281e493b